pkg/network/ovn_kubernetes.go

```diff
@@ -1302,15 +1305,19 @@ func getTargetInterConnectZoneMode(kubeClient cnoclient.Client) (targetZoneModeT
 		targetZoneMode.zoneMode = zoneModeMultiZone
 	}
 
-	if temporaryFromConfigMap, ok := interConnectConfigMap.Data["temporary"]; ok {
-		targetZoneMode.temporary = strings.ToLower(temporaryFromConfigMap) == "true"
-	}
-
 	if _, ok := interConnectConfigMap.Data["ongoing-upgrade"]; ok {
 		targetZoneMode.ongoingUpgrade = true
 	}
 
-	klog.Infof("target zone mode: %+v", targetZoneMode)
+	if _, ok := interConnectConfigMap.Data["fast-forward-to-multizone"]; ok {
+		targetZoneMode.fastForwardToMultiZone = true
+		if targetZoneMode.zoneMode != zoneModeMultiZone {
+			klog.Warningf("Forcing interconnect zone mode to multizone due to 'fast-forward-to-multizone' being set")
+			targetZoneMode.zoneMode = zoneModeMultiZone
```

Comment:
so this is sort of a disruptive change in case ongoingUpgrade=true and someone sets this right? example, single zone is rolling out and stuck and now we set `fastForwardToMultiZone` to true to unblock and move forward with multizone.. will the single zone pods be left behind or will cleanup happen? 

Also I guess we can add something to the PR description or enhancement update around this option to say "at your own risk - to be used only with guidance from eng or something"

Reply:
Yes, using the new flag is disruptive: CNO will push the final multizone YAMLs regardless of where the cluster is during the 4.13->4.14 upgrade. After this PR merges, my idea was to write a document about this for support. Either that or I update the enhancement...

As for the single-zone pods, they will be removed as a consequence of the new ones being pushed.